### ISSUE TITLE
Is there/should there be a keyboard command to end keyboard nav?

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#205]
For https://github.com/phetsims/QA/issues/419
When zooming in you can use the arrow keys to scroll around.  However, if you are using keyboard nav, this doesn't work anymore.  You can still scroll using the track pad if there is one, but many mouses only allow scrolling up and down and a track pad may not be available.  In order to scroll using the arrows you need to exit keyboard nav.  The only way I can find to do this is click with the mouse.  While the screen does generally follow the keyboard focus, if you open a dialog like the light spectrum or the keyboard nav help, only a corner is visible and you need to scroll to see the rest.

**Comment by jessegreenberg** [comment]
Discussed with @terracoda - we decided that since this is not a standard feature it is not expected and therefore not worth adding at this time. There were some thoughts that this could create a keyboard focus trap if used incorrectly as well. We are not going to include at this time. The other issue mentioned here (unable to pan while focus is on the dialog close button) will be addressed in other issues open for phetsims/QA#419.
